cmd/manager/main.go

```diff
@@ -128,6 +135,10 @@ func main() {
 		log.Error(err, "")
 		os.Exit(1)
 	}
+	// start cache and wait for sync
+	cache := mgr.GetCache()
+	go cache.Start(stopCh)
+	cache.WaitForCacheSync(stopCh)
```

Comment:
I'm curious on this, since the cache isn't populated until the controllers are initialized I have used in the past a client from the controller-runtime. This is great, did you find this in the documentation or do some investigating?

Reply:
This is a standard pattern in non sdk based controllers. I just knew it exists :) 
What is of interest is that it does not start properly now in v4 cluster as the cache is not started. and it hands for ~30 min for next sync cycle. But I was not able to find what changed